### PR TITLE
EVAKA 3795 attendances mobile POC

### PR DESCRIPTION
**Pull request by massproduction**
#### Summary
Mobile POC for adding attendances or absences for children.

Spec here: https://www.figma.com/proto/UyFTAPt5kFhErtP56e7Ze8/Evaka-design?node-id=508%3A84&scaling=scale-down

### CODE REVIEW

File: frontend/packages/employee-frontend/src/api/absences.ts [hunk 1]
```diff
@@ -84,3 +86,14 @@ export async function postStaffAttendance(
     .then((res) => Success(res.data))
     .catch(Failure)
 }
+
+export async function postChildAttendance(
```
Comment (Joosakur) [inline frontend/packages/employee-frontend/src/api/absences.ts:90]: isnt this postAbsence?

**Comment by Joosakur** [comment]
Got some more minor comments about the code but will save them for now since this is just a PoC.

File: frontend/packages/employee-frontend/package.json [hunk 2]
```diff
@@ -46,6 +46,7 @@
     "react-day-picker": "^7.4.0",
     "react-dom": "^16.12.0",
     "react-focus-lock": "2.4.1",
+    "react-meta-tags": "0.7.4",
```
Comment (Joosakur) [inline frontend/packages/employee-frontend/package.json:49]: is the licence for this ok?

**Review by Joosakur — APPROVED** [review]

File: frontend/packages/employee-frontend/package.json (hunk 2 above)
Comment (kustaahuhtala) [reply frontend/packages/employee-frontend/package.json:49]: It's MIT, so all good.

File: frontend/packages/employee-frontend/src/api/absences.ts (hunk 1 above)
Comment (massproduction) [reply frontend/packages/employee-frontend/src/api/absences.ts]: Fixed

**Pull request merged**